storage/tianmu/loader/load_parser.cpp

```diff
@@ -29,44 +29,44 @@ namespace Tianmu {
 namespace loader {
 LoadParser::LoadParser(RCAttrPtrVect_t &attrs, const system::IOParameters &iop, uint packsize,
                        std::unique_ptr<system::Stream> &f)
-    : attrs(attrs),
-      start_time(types::RCDateTime::GetCurrent().GetInt64()),
-      ioparam(iop),
-      pack_size(packsize),
-      rejecter(packsize, iop.GetRejectFile(), iop.GetAbortOnCount(), iop.GetAbortOnThreshold()),
-      no_obj(attrs[0]->NumOfObj()) {
+    : attrs_(attrs),
+      start_time_(types::RCDateTime::GetCurrent().GetInt64()),
+      ioparam_(iop),
```

Comment:
as the style of `start_time` and `pack_size`.  suggest: `io_param_`. and `no_obj_` changes to `num_of_obj_`

Reply:
The modification has been completed